README.md

```diff
@@ -48,13 +48,13 @@ go run build/make.go --all-platforms
 After compilation
 
 ```
-go run build/make.go --install
```

Comment:
`go` options are not the same as `gauge` 
So here it's still `go run build/make.go --install`